src/EnergyPlus/HVACManager.cc

```diff
@@ -2400,8 +2398,9 @@ void ReportAirHeatBalance(EnergyPlusData &state)
         auto &zone = state.dataHeatBal->Zone(ZoneLoop);
         auto &znAirRpt = state.dataHeatBal->ZnAirRpt(ZoneLoop);
         auto &zoneEquipConfig = state.dataZoneEquip->ZoneEquipConfig(ZoneLoop);
-        auto &refDoorMixing = state.dataHeatBal->RefDoorMixing(ZoneLoop);
```

Comment:
This right here caused Linux debug failures for files without refrigeration door mixing. @Myoldmopar found it on a linux debug build and execution of a failed file. A debug build with all exceptions on also found it on Windows. Moved this inside `if (state.dataHeatBal->TotRefDoorMixing > 0)` and all is good. I also moved the reference to class for MixSenLoad and MixLatLoad while in here.